.github/workflows/crowdin-download.yml

```diff
@@ -0,0 +1,102 @@
+name: Crowdin Download
+on:
+  workflow_dispatch:
+  # schedule:
+  #   # runs every day at 12:00 noon UTC
+  #   - cron: '0 12 * * *'
+
+jobs:
+  i18n-download:
+    name: Download
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout Source Files
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
+
+      - name: Generate Crowdin Config
+        uses: freecodecamp/crowdin-action@main
+        env:
+          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
+          CROWDIN_API_URL: "https://freecodecamp.crowdin.com/api/v2/"
+          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_DOCS }}
+          PLUGIN: "generate-config"
+          PROJECT_NAME: "rpg"
+
+      ##### Download Chinese #####
+      - name: Crowdin Download Chinese Translations
+        uses: crowdin/github-action@master
+        # options: https://github.com/crowdin/github-action/blob/master/action.yml
+        with:
+          # uploads
+          upload_sources: false
+          upload_translations: false
+          auto_approve_imported: false
+          import_eq_suggestions: false
+
+          # downloads
+          download_translations: true
+          download_language: zh-CN
+          skip_untranslated_files: false
```

Comment:
Why is this `false`?

Reply:
Because if the english files are updated, we want to pull down the changes for the i18n versions whether they're fully translated or not.